$MirOS: ports/games/bsdgames/patches/patch-rogue_message_c,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- rogue/message.c.orig	Thu Jan  1 00:00:00 1970
+++ rogue/message.c	Thu Feb 22 21:51:08 2007
@@ -109,9 +109,12 @@ messagef(boolean intrpt, const char *fmt
 	va_list ap;
 	char buf[DCOLS];
 
-	va_start(ap, fmt);
-	vsnprintf(buf, sizeof(buf), fmt, ap);
-	va_end(ap);
+	if (fmt) {
+		va_start(ap, fmt);
+		vsnprintf(buf, sizeof(buf), fmt, ap);
+		va_end(ap);
+	} else
+		buf[0] = '\0';
 
 	message(buf, intrpt);
 }
